src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $white;
+  border: 1px solid $gray-elements;
```

Comment:
Ensure that the SCSS variable `$gray-elements` is defined in your `variables.scss` file or imported correctly. This is crucial for maintaining styling consistency.